<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icon-small.png</normaloff>images/icon-small.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="1" alignment="Qt::AlignRight|Qt::AlignBottom">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>icon.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Stress</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <property name="labelAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="formAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <item row="0" column="0" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QFormLayout" name="formLayout_8">
                   <property name="fieldGrowthPolicy">
                    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                   </property>
                   <item row="0" column="0">
                    <widget class="QComboBox" name="comboBox">
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <item>
                      <property name="text">
                       <string>Pa (N/m²)</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>MPa</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit"/>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QFormLayout" name="formLayout_9">
                   <item row="0" column="0">
                    <widget class="QComboBox" name="comboBox_2">
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <item>
                      <property name="text">
                       <string>kN/m²</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>N/mm²</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit_2"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QComboBox" name="comboBox_3">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>Kg/mm²</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Kg/m²</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_4"/>
               </item>
               <item row="2" column="0">
                <widget class="QComboBox" name="comboBox_4">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>psf</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>psi</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEdit_9"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Luminance</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QFormLayout" name="formLayout_4">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <property name="labelAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="formAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>cd/m²</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_6"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_8"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Illuminance</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <property name="labelAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="formAlignment">
                <set>Qt::AlignCenter</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Lux (lx)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_5"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_7"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Length</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QFormLayout" name="formLayout_5">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <property name="formAlignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="text">
                  <string>Metres (m)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_3"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QFormLayout" name="formLayout_6">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_12">
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Feet (ft)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineEdit_12"/>
                 </item>
                </layout>
               </item>
               <item row="0" column="1">
                <layout class="QFormLayout" name="formLayout_7">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_13">
                   <property name="minimumSize">
                    <size>
                     <width>46</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Inches (in)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineEdit_13"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" rowspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1" rowspan="2">
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit">
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Here is a clipboard...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionQuote"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionQuote">
   <property name="text">
    <string>Quote</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
